packages/enzyme-test-suite/package.json

```diff
@@ -55,4 +55,4 @@
     "eslint-plugin-react": "^7.11.1",
     "react-is": "^16.7.0"
   }
-}
+}
```

Comment:
please revert this so that all files have trailing newlines

Reply:
Whoops. Didn't mean to `git add` this.